access/resource_ip_access_list_test.go

```diff
@@ -117,37 +118,43 @@ func TestIPACLUpdate(t *testing.T) {
 		Fixtures: []qa.HTTPFixture{
 			{
 				Method:   http.MethodGet,
-				Resource: "/api/2.0/ip-access-lists/" + TestingID,
-				Response: IpAccessListStatusWrapper{
-					IPAccessList: IpAccessListStatus{
-						ListID:        TestingID,
-						Label:         TestingLabel,
-						ListType:      TestingListType,
-						IPAddresses:   TestingIPAddresses,
-						AddressCount:  2,
-						CreatedAt:     87939234,
-						CreatorUserID: 1234556,
-						UpdatedAt:     87939234,
-						UpdatorUserID: 1234556,
-						Enabled:       TestingEnabled,
+				Resource: "/api/2.0/ip-access-lists/" + TestingId + "?",
+				Response: settings.CreateIpAccessListResponse{
+					IpAccessList: &settings.IpAccessListInfo{
+						ListId:       TestingId,
+						Label:        TestingLabel,
+						ListType:     TestingListType,
+						IpAddresses:  TestingIpAddresses,
+						AddressCount: 2,
+						CreatedAt:    87939234,
+						CreatedBy:    1234556,
+						UpdatedAt:    87939234,
+						UpdatedBy:    1234556,
+						Enabled:      TestingEnabled,
 					},
 				},
 			},
 			{
-				Method:   http.MethodPut,
-				Resource: "/api/2.0/ip-access-lists/" + TestingID,
-				Response: IpAccessListStatusWrapper{
-					IPAccessList: IpAccessListStatus{
-						ListID:        TestingID,
-						Label:         TestingLabel,
-						ListType:      TestingListType,
-						IPAddresses:   TestingIPAddresses,
-						AddressCount:  2,
-						CreatedAt:     87939234,
-						CreatorUserID: 1234556,
-						UpdatedAt:     87939234,
-						UpdatorUserID: 1234556,
-						Enabled:       TestingEnabled,
+				Method:   http.MethodPatch,
```

Comment:
Put -> Patch because: https://docs.databricks.com/api-explorer/workspace/ipaccesslists/update update uses PATCH.  Can this be breaking for customers? Also why was put using query parameters earlier (wasn't taking expected request)? Should put use body?